pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -295,27 +295,27 @@ func (e *TokensController) syncSecret() {
 			if err := clientretry.RetryOnConflict(RemoveTokenBackoff, func() error {
 				return e.removeSecretReference(secretInfo.namespace, secretInfo.saName, secretInfo.saUID, secretInfo.name)
 			}); err != nil {
-				klog.Error(err)
+				klog.ErrorS(err, "Error removing secret", "serviceaccount", klog.KRef(secretInfo.namespace, secretInfo.saName), "secret", klog.KRef("", secretInfo.name))
```

Comment:
null secret namespace?

Reply:
I think in this `case`, sa and secret are in the same namespace, do you think it needs to be added?

Reply:
May be it needs.

Reply:
done.